src/picongpu/include/fields/currentDeposition/Esirkepov/Esirkepov.hpp

```diff
@@ -136,12 +136,9 @@ struct Esirkepov<T_ParticleShape, DIM3>
          * If previous position was greater than current position we change our interval
          * from [begin,end) to [begin+1,end+1).
          */
-        /** \todo -(-int(bool)) is a workaround for a nvcc bug
```

Comment:
puh... that would be interesting to be tested with both `cuda 5.5` and `cuda 6.5`